argocd/resource_argocd_application_set_test.go

```diff
@@ -3322,9 +3322,9 @@ resource "argocd_application_set" "custom_namespace" {
       spec {
 		project = argocd_project.custom_namespace.metadata[0].name
 		source {
-		  repo_url        = "https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami"
-		  chart           = "redis"
-		  target_revision = "16.9.11"
+		  repo_url        = "https://kubernetes-sigs.github.io/descheduler"
```

Comment:
Descheduler is a zero-dependency thing from kubernetes-sigs and hopefully around for a long time. It's deployed as a cronjob by default so deployment is fast.